code/go/0chain.net/smartcontract/stakepool/stakepool.go

```diff
@@ -91,10 +91,11 @@ type DelegatePoolStat struct {
 	ProviderId   string          `json:"provider_id"`   // id
 	ProviderType spenum.Provider `json:"provider_type"` // ype
 
-	TotalReward  currency.Coin `json:"total_reward"`
-	TotalPenalty currency.Coin `json:"total_penalty"`
-	Status       string        `json:"status"`
-	RoundCreated int64         `json:"round_created"`
+	TotalReward  currency.Coin    `json:"total_reward"`
+	TotalPenalty currency.Coin    `json:"total_penalty"`
+	Status       string           `json:"status"`
+	RoundCreated int64            `json:"round_created"`
+	StakedAt     common.Timestamp `json:"staked_at"`
```

Comment:
This new added field caused the lint error, you'll need to install msgp and run msgp with command in project root:
```sh
make install-msgp
make msgp
```

Reply:
done